engine/table/src/main/java/io/deephaven/stream/StreamToBlinkTableAdapter.java

```diff
@@ -68,11 +75,11 @@ public class StreamToBlinkTableAdapter extends ReferenceCountedLivenessNode
     private ChunkColumnSource<?>[] prevChunkSources;
 
     /** A list of failures that have occurred. */
-    private List<Throwable> enqueuedFailure;
+    private List<Throwable> enqueuedFailures;
```

Comment:
This could be final and allocated in the constructor, and instead the checkers could check isEmpty() instead of null.